desc/basis.py

```diff
@@ -657,57 +657,19 @@ def evaluate(self, grid, derivatives=np.array([0, 0, 0]), modes=None):
         _, t, z = grid.nodes.T
         _, m, n = modes.T
 
-        t = t[tidx]
-        z = z[zidx]
+        t = kwargs["t"] if "t" in kwargs else t[tidx]
```

Comment:
`kwargs.get` would be cleaner here

Reply:
then both branches would execute regardless